backend/src/tasks/test/wappalyzer.test.ts

```diff
@@ -139,11 +130,8 @@ describe('wappalyzer', () => {
     await handler(commandOptions);
     scope.done();
     expect(wappalyzer).toHaveBeenCalledTimes(2);
-    expect(saveDomainsToDbMock).toHaveBeenCalledTimes(1);
```

Comment:
We should update this test to actually check that the domains' `webTechnologies` attribute was actually updated (this might require creating actual domains beforehand, rather than just creating mock domain objects)

Reply:
@epicfaace Do you think you could make this change today? If not I'm fine merging this in without for now.

Reply:
Yes